instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/patch.py

```diff
@@ -262,6 +406,91 @@ def _set_response_attributes(
         )
 
 
+def _set_embeddings_response_attributes(
+    span,
+    result,
+    event_logger: EventLogger,
+    capture_content: bool,
+    input_text: str,
+):
+    """Set attributes on the span based on the embeddings response."""
+    # Set the model name if available
+    set_span_attribute(
+        span, GenAIAttributes.GEN_AI_RESPONSE_MODEL, result.model
+    )
+
+    # Set embeddings dimensions if we can determine it from the response
+    if getattr(result, "data", None) and len(result.data) > 0:
+        first_embedding = result.data[0]
+        if getattr(first_embedding, "embedding", None):
+            set_span_attribute(
+                span,
+                "gen_ai.embeddings.dimensions",
+                len(first_embedding.embedding),
+            )
+
+    # Get the usage
+    if getattr(result, "usage", None):
+        set_span_attribute(
+            span,
+            GenAIAttributes.GEN_AI_USAGE_INPUT_TOKENS,
+            result.usage.prompt_tokens,
+        )
+        set_span_attribute(
+            span,
+            GenAIAttributes.GEN_AI_USAGE_OUTPUT_TOKENS,
+            result.usage.total_tokens,
```

Comment:
total is not output, right? we should not have any output tokens for embeddings if I recall correctly

Reply:
Yes, we have only input tokens in semconv